x-pack/plugins/upgrade_assistant/public/application/components/constants.tsx

```diff
@@ -26,6 +26,12 @@ export const DEPRECATION_TYPE_MAP = {
   ml_settings: i18n.translate('xpack.upgradeAssistant.esDeprecations.mlDeprecationTypeLabel', {
     defaultMessage: 'Machine Learning',
   }),
+  health_indicator: i18n.translate(
+    'xpack.upgradeAssistant.esDeprecations.healthIndicatorTypeLabel',
+    {
+      defaultMessage: 'Health Indicator',
```

Comment:
Should this be sentence-case?

```suggestion
      defaultMessage: 'Health indicator',
```

Reply:
These are title labels, following the other i18n labels here they are all capitalized like this